saleor/graphql/shop/mutations.py

```diff
@@ -386,7 +388,10 @@ class Arguments:
         )
 
     class Meta:
-        description = "Update shop order settings."
+        description = (
+            "Update shop order settings."
+            f"{DEPRECATED_IN_3X_INPUT} Use `channelUpdate` instead."
```

Comment:
We have specific label for mutation. `DEPRECATED_IN_3X_MUTATION`

Reply:
We should also mention, that calling this mutation will update all values across all channels